backend/core/forms.py

```diff
@@ -22,3 +22,28 @@ def clean(self):
             raise ValidationError("Choose one, and only one, filter to apply")
 
         return data
+
+
+class BulkAddUsersToTeamForm(forms.Form):
+    email_addresses = forms.CharField(
+        label="Email addresses",
+        widget=forms.Textarea(attrs={"rows": 15, "style": "display: flex; width:100%"}),
+        help_text="Emails can be separated by commas, spaces and newlines.",
+        required=True,
+        empty_value=False,
+    )
+
+    def clean_email_addresses(self):
+        email_addresses_str = self.cleaned_data["email_addresses"]
+        if not email_addresses_str:
+            raise ValidationError(
+                f"No email addresses entered.",
+            )
+        email_addresses_list = self.get_email_addresses_from_str(email_addresses_str)
+        return email_addresses_list
```

Comment:
I need to insist here: Form validation needs to happen all the way. 

If any of the email addresses are not associated with a User then we need to raise a ValidationError that says which email addresses are causing problems.  This means we need the form to do a database query.

We should not have to query the database again in the view. So the form instance should maintain a list of users. 

There are a few ways to do this:
- You can just use something like `self.users` (messy because it's non-standard, but it'll work)
- get `clean_email_addresses` to return the users.  (messy because names matter)
- maybe you can make a hidden field called users and then have "cleaned_data['users']" get populated from the other field. This seems like it will be the tidiest but I haven't tried this before.